gpytorch/lazy/cat_lazy_tensor.py

```diff
@@ -364,15 +364,24 @@ def devices(self):
     def device_count(self):
         return len(set(self.devices))
 
-    def to(self, device_id):
+    def to(self, device=None, dtype=None):
         """
-        returns a new CatLazyTensor with device_id as the output_device
+        returns a new CatLazyTensor with device as the output_device
         Warning: this does not move the LazyTensors in this CatLazyTensor to
-        device_id
+        device
         """
+        if type(device) is torch.dtype and dtype is None:
+            dtype = device
+            device = self.output_device
```

Comment:
This is kind of funky. Is this what pytorch does?

Reply:
Not quite, it's left as `*args, **kwargs` from what I can tell here: https://pytorch.org/docs/master/generated/torch.Tensor.to.html#torch.Tensor.to . I could replace with that type of inferrring instead?